src/components/category/category.html

```diff
@@ -19,7 +19,7 @@
                       <br>
                       <br>
                       <span v-for="(cat, index) in item.categories" class="font-weight-bold textcolor">
-                        <router-link class="font-weight-bold linkclass" :to="'/categories/'+cat.slug">{{cat.name.charAt(0).toUpperCase() + cat.name.slice(1)}}</router-link>
+                        <router-link class="font-weight-bold linkclass" :to="'/categories/'+cat.slug">{{cat.name | capitalize}}</router-link>
```

Comment:
if we have `text-uppercase` class in vuetify.css file, why you had created this filter.

Reply:
Because we need to display only the first character of the word in uppercase. With the help of text-uppercase class it uppercase all the character, so that's wrong.